jina/parsers/peapods/pod.py

```diff
@@ -64,12 +64,8 @@ def mixin_base_pod_parser(parser):
     )
     gp.add_argument(
         '--peas-hosts',
-        action=KVAppendAction,
-        metavar='KEY: VALUE',
-        nargs='*',
-        help='''The hosts of the peas when parallel greater than 1,
-        pea have a new host address if the pea_id present in the map.
-        otherwise pea host will be identical to the host of pod.
-        Represented as a key value pair in argument.
-        key is the pea_id, and value is the host address.''',
+        nargs='+',
```

Comment:
Please add a `type` to the arg to get rid of the below error. Pydantic models are based on these types.
```
<class 'pydantic.errors.ConfigError'>
unable to infer type for attribute "peas_hosts"
```

Reply:
`type=list` is not what we want. This will generate a list of lists.